.github/workflows/automerge.yml

```diff
@@ -0,0 +1,144 @@
+name: automerge
+
+on:
+  workflow_run:
```

Comment:
Another nice thing we can do with `workflow_run` events is that we can use them to remove https://github.com/Homebrew/homebrew-core/labels/CI-long-timeout labels from PRs when CI finishes.

Using that to find another https://github.com/Homebrew/homebrew-core/labels/long%20build PR and re-running their CI is slightly more complicated, but may be possible.